modules/kubernetes-role-assignment.adoc

```diff
@@ -3,13 +3,13 @@
 // * operating/review-cluster-configuration.adoc
 :_mod-docs-content-type: PROCEDURE
 [id="kubernetes-role-assignment_{context}"]
-= Finding Kubernetes roles are their assignment
+= Finding Kubernetes roles and their assignment
 
 [role="_abstract"]
-Use the *Configuration Management* view to get information about which Kubernetes roles are assigned to specific users and groups.
+Use the *Configuration Management* view to get information about the Kubernetes roles that are assigned to specific users and groups.
 
 .Procedure
-. Navigate to the RHACS portal and click *Configuration Management* from the left-hand navigation menu.
-. Select *RBAC Visibility and Configuration* -> *Users and Groups* from the header in the *Configuration Management* view.
+. Navigate to the {product-title-short} portal and click *Configuration Management*.
```

Comment:
```suggestion
. Go to to the {product-title-short} portal and click *Configuration Management*.
```
Per [the ISG guidance on "navigate"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n): "Use only when a more direct verb, such as click, select, browse, or go to, is not applicable."

Reply:
Unfortunately, we have a couple hundred instances of "navigate" in our docs (existing before RH acquisition). I'll make a ticket to update them all instead of piece by piece.